forge-gui/res/cardsfolder/upcoming/hourglass_of_the_lost.txt

```diff
@@ -0,0 +1,8 @@
+Name:Hourglass of the Lost
+ManaCost:2 W
+Types:Artifact
+A:AB$ Mana | Cost$ T | Produced$ W | SubAbility$ DBPutCounter | SpellDescription$ Add {W}. Put a time counter on CARDNAME.
+SVar:DBPutCounter:DB$ PutCounter | CounterType$ TIME | CounterNum$ 1
+A:AB$ ChangeZoneAll | Cost$ T SubCounter<X/TIME> Exile<1/CARDNAME> | ChangeType$ Permanent.YouOwn+nonLand+cmcEQX | Origin$ Graveyard | Destination$ Battlefield | SorcerySpeed$ True | SpellDescription$ Return each nonland permanent card with mana value X from your graveyard to the battlefield. Activate only as a sorcery.

```

Comment:
Hmm
<img width="294" alt="image" src="https://github.com/Card-Forge/forge/assets/103371817/2a3172d3-9424-4ae9-9865-70d037404a20">
+ `CostDesc$ {T}, Remove X time counters from CARDNAME and exile it:`